addons/api/package.json

```diff
@@ -41,6 +40,7 @@
     "@babel/plugin-proposal-decorators": "^7.18.6",
     "@ember/optional-features": "^2.0.0",
     "@ember/test-helpers": "^2.8.1",
+    "@embroider/test-setup": "^0.48.1",
```

Comment:
`@embroider/test-setup` is using at development time (`ember-try`) not in production. That's why I think makes more sense to have it as a `devDependency` than as `dependency`.